### PR TITLE
ROC-5033: Add court determination Instalments/Instalments scenario

### PR DESCRIPTION
**Pull request by KiranVarma02**
### JIRA link
https://tools.hmcts.net/jira/browse/ROC-5033

### Change description
Added court determination instalments/instalments scenario. 

Once overnight builds are setup we have to move all claimant response journeys to overnight except instalments/instalments scenario

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/integration-test/tests/citizen/claimantResponse/full_admission_test.ts [hunk 1]
```diff
@@ -55,7 +55,7 @@ if (process.env.FEATURE_ADMISSIONS === 'true') {
     I.see('You’ve signed a settlement agreement')
   })
 
-  Scenario('I can as a claimant accept the defendants full admission by set date with settlement agreement and rejecting defendants payment method in favour of immediate payment @overnight @admissions', async (I: I) => {
+  Scenario('I can as a claimant accept the defendants full admission by set date with settlement agreement and rejecting defendants payment method in favour of immediate payment @@nightly @admissions', async (I: I) => {
```
Comment (markry-kainos) [inline src/integration-test/tests/citizen/claimantResponse/full_admission_test.ts]: double `@` (few more in file too :) )

File: package.json [hunk 2]
```diff
@@ -14,6 +14,7 @@
     "test:routes-parallel": "TS_NODE_TRANSPILE_ONLY=true NODE_ENV=mocha LOG_LEVEL=OFF mocha-parallel-tests --opts mocha.opts $(find src/test \\( -name '*.ts' \\) -path '*/routes/*') --timeout 10000 --reporter-options reportFilename=routes,inlineAssets=true,reportTitle=citizen-frontend-routes",
     "test:a11y": "TS_NODE_TRANSPILE_ONLY=true NODE_ENV=mocha LOG_LEVEL=ERROR mocha --exit --opts mocha.opts src/test/a11y/a11y.ts --timeout 60000 --reporter-options reportFilename=a11y,inlineAssets=true,reportTitle=citizen-frontend-a11y",
     "test:functional": "./bin/run-functional-tests.sh",
+    "test:functional-nightly": "./bin/run-nightly-tests.sh",
```
Comment (markry-kainos) [inline package.json]: `test:fullfunctional` as per: https://github.com/hmcts/cnp-jenkins-library/blob/38da6b13a6affa5e6bf2d9a54ec7f43347eb672b/src/uk/gov/hmcts/contino/YarnBuilder.groovy#L67

File: src/integration-test/tests/citizen/claimantResponse/full_admission_test.ts (hunk 1 above)
Comment (KiranVarma02) [reply src/integration-test/tests/citizen/claimantResponse/full_admission_test.ts]: done

File: package.json (hunk 2 above)
Comment (KiranVarma02) [reply package.json]: done

**Pull request merged**